### DISCUSSION TITLE
Bug Report:

### DISCUSSION THREAD

**Discussion by barretin** [KadeArchive/Kade-Engine#959] [Show and tell]
### If you are playing a downloaded version of the game, what operating system are you using?
Windows (`x86`), Windows (`x86_64`), Linux, or macOS? Specify below.

 Windows (`x86_64`)

## What version of the game are you using? Look in the bottom left corner of the main menu.

1.4.2

## Have you identified any steps to reproduce the bug? If so, please describe them below. Use images if possible.

no

## Please describe your issue. Provide extensive detail and images if possible.

The arrows doesn't count even if i press them in the correct time, making impossible to play

**Comment by puyoxyz** [comment]
set safe frames to 10
